js/index.js

```diff
@@ -40,3 +41,93 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let ctaImg = document.getElementById('cta-img');
+ctaImg.setAttribute('src', siteContent['cta']['img-src']);
+
+let middleImg = document.getElementById('middle-img');
+middleImg.setAttribute('src', siteContent['main-content']['middle-img-src']);
+
+let nav = document.getElementsByTagName('nav')[0];
+// create new anchor to append to the nav bar
+let newNavLink = document.createElement('a');
+nav.appendChild(newNavLink);
+
+
+// Update the header children
+let navLinks = document.getElementsByTagName('a');
+for(let i = 0; i < navLinks.length; i++) {
+  let navItem = navLinks.item(i);
+  let innerText = 'nav-item-' + (i + 1);
+  navItem.innerText = siteContent['nav'][innerText];
+  navItem.style.color = 'green';
+}
+
+
+
+//****** CTA section
+// HEADER
+let header = document.getElementsByTagName('h1')[0];
+let headerText = siteContent['cta']['h1'];
+headerText = headerText.split(' ').join('<br>');
+header.innerHTML = headerText;
+// BUTTON
+let getStartedButton = document.getElementsByTagName('button')[0];
+getStartedButton.innerText = siteContent['cta']['button'];
+
+//******* Main content
+
+let textContentElements = document.querySelectorAll('.text-content'),
+        featuresElement = textContentElements[0],
+           aboutElement = textContentElements[1],
+        servicesElement = textContentElements[2],
+         productElement = textContentElements[3],
+          visionElement = textContentElements[4]
+            mainContent = siteContent['main-content'];
+
+//******** Features Content
+// H4
+featuresElement.children.item(0).innerText = mainContent['features-h4'];
+// P
+featuresElement.children.item(1).innerText = mainContent['features-content'];
+
+//******** About Content
+// H4
+aboutElement.children.item(0).innerText = mainContent['about-h4'];
+// P
+aboutElement.children.item(1).innerText = mainContent['about-content'];
+
+//******** Services Content
+// H4
+servicesElement.children.item(0).innerText = mainContent['services-h4'];
+// P
+servicesElement.children.item(1).innerText = mainContent['services-content'];
+
+//******** Product Content
+// H4
+productElement.children.item(0).innerText = mainContent['product-h4'];
+// P
+productElement.children.item(1).innerText = mainContent['product-content'];
+
+//******** Vision Content
+// H4
+visionElement.children.item(0).innerText = mainContent['vision-h4'];
+// P
+visionElement.children.item(1).innerText = mainContent['vision-content'];
+
+//******** Contact Section
+let contactContent = siteContent['contact'];
+let contactSection = document.getElementsByClassName('contact');
+let contactChildren = contactSection[0].children;
+// H4
+contactChildren.item(0).innerText = contactContent['contact-h4'];
+// address
+contactChildren.item(1).innerText = contactContent['address'];
+// phone number
```

Comment:
It's more common to see `item[0]` instead of `item(0)`. 
`()` implies that it's a function, whereas `[]` implies that it's an index.